analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Sonar/SonarExplodedGraphTest.cs

```diff
@@ -901,6 +900,249 @@ public void ExplodedGraph_IsPattern_WithPositionalPattern()
             context.WalkWithInstructions(4);
         }
 
+        [TestMethod]
```

Comment:
Testing "just one" scenario should be enough. We don't want to invest in this part of codebase.

Reply:
Or one long test with all unsupported cases that will just prove that it doesn't fail.